internal/state/validation_test.go

```diff
@@ -5,14 +5,15 @@ import (
 	"testing"
 	"time"
 
+	"github.com/tendermint/tendermint/crypto/ed25519"
+	"github.com/tendermint/tendermint/crypto/tmhash"
```

Comment:
nit: order these below with the rest of the tendermint library imports.

Reply:
having go imports setup in your editor should take care of this sort of thing.

Reply:
It looks like this happens if you run `goimports` on a file where the import ordering was previously incorrect. It doesn't realize which block it should put the incorrectly ordered libraries into so it just puts them into a new block but generally it definitely should be the way to avoid this.